Avoid:
    fatal: not a git repository (or any of the parent directories): .git

See https://github.com/emscripten-core/emscripten/pull/18441/

--- tools/install.py.orig	2021-03-06 00:46:36.000000000 -0700
+++ tools/install.py	2021-03-24 17:27:28.000000000 -0700
@@ -95,7 +95,10 @@ def main():
   logging.basicConfig(level=logging.DEBUG if args.verbose else logging.INFO)
   os.makedirs(target)
   copy_emscripten(target)
-  add_revision_file(target)
+  if os.path.isdir('.git'):
+    # Add revision flag only if the source directory is a Git repository
+    # and not a source archive
+    add_revision_file(target)
   return 0
 
 
